Dockerfile

```
@@ -1,49 +1,39 @@
 FROM ros:noetic-ros-core
```

```suggestion
FROM ros:noetic-ros-base
```

According to [source dockerfile](https://github.com/osrf/docker_images/blob/master/ros/noetic/ubuntu/focal/ros-base/Dockerfile) of noetic-ros-base we wouldn't have to install rosdep separately with this base image.

---

I see it was changes. I'm resolving this thread.